mscore/file.cpp

```diff
@@ -1602,6 +1602,7 @@ void MuseScore::exportFile()
 #endif
       fl.append(tr("Standard MIDI File") + " (*.mid)");
       fl.append(tr("MusicXML File") + " (*.xml)");
+      fl.append(tr("MusicXML 3.1+ File") + " (*.musicxml)");
```

Comment:
Unless the user changes the extension to .xml manually or upgrade his other software. Indeed. But let's live in the future and pave the way ;)
In general, mxl is quite well supported too btw.